src/board.cpp

```diff
@@ -40,15 +40,16 @@ void Board::swap(pint one1, pint one2, pint two1, pint two2) {
     (*this)(two1, two2) = temp;
 }
 
-void Board::eraseCombo(VisitedIndex* visited, Combo& combo, pint ox, pint oy) {
+void Board::eraseCombo(VisitedIndex* visited,
+                       std::deque<pint>& queue,
+                       Combo& combo,
+                       pint ox,
+                       pint oy) {
     // assume this index is valid
```

Comment:
If anything is wrong, this function is probably the cause. It is just too fast and I am sure it is not right.

Reply:
This function works but stl makes it very heavy